plugin/builtin/patchjson6902transformer/PatchJson6902Transformer.go

```diff
@@ -81,6 +81,11 @@ func (p *plugin) Transform(m resmap.ResMap) error {
 	if err != nil {
 		return err
 	}
+
+	if len(resources) == 0 {
```

Comment:
This is the correct behavior!

Reply:
Thank you for your review. I will keep patchjson6902transformer changes as it is. I will work on other suggestions.